api/models.py

```diff
@@ -1,16 +1,42 @@
-from django.core.validators import EmailValidator
 from django.db import models
 from api.logistics.choice_enums import YEAR_IN_SCHOOL_CHOICES, LEARNING_GOALS_CHOICES, EXPERIENTIAL_LEARNING_HOURS_TYPES
 from api.reliability.validators import no_future_dates, hour_instance_validator, minutes_validator, student_id_validator
 
 from auth_backend.modules.common import models as common_models
+from auth_backend.modules.user.models import BaseVologUser
+
+
+class Mentor(models.Model):
+    #I added the foreign key as we have two users, student and mentor. Foreign key is used to distigush which user is which. 
```

Comment:
I don't think this is somewhere to use foreign keys. I met with Prof. Meng after cybersecurity to ask for advice regarding the two approaches (inheritance vs relations) and his stance was that it makes more sense to just do inheritance here.

Reply:
done as requested